apps/melding-form/src/app/(map)/locatie/kies/_components/Map/utils/updateAssetLayer.test.tsx

```diff
@@ -0,0 +1,78 @@
+import L from 'leaflet'
+import { MutableRefObject } from 'react'
+import { Mock } from 'vitest'
+
+import { addAssetLayerToMap } from './addAssetLayerToMap'
+import { fetchAssets } from './fetchAssets'
+import { updateAssetLayer } from './updateAssetLayer'
+import { ASSET_ZOOM_THRESHOLD } from './updateAssetLayer'
+import { containerAssets } from 'apps/melding-form/src/mocks/data'
+
+vi.mock('./fetchAssets', () => ({
+  fetchAssets: vi.fn(() => ({ features: containerAssets })),
+}))
+
+vi.mock('./addAssetLayerToMap', () => ({
+  addAssetLayerToMap: vi.fn(),
+}))
+
+const mapInstanceMock = {
+  getZoom: vi.fn().mockImplementation(() => ASSET_ZOOM_THRESHOLD),
+  getSize: vi.fn(() => ({ x: 800, y: 600 })),
+} as unknown as L.Map
+
+const mockDefaultValues = {
+  assetLayerRef: { current: null } as MutableRefObject<L.Layer | null>,
+  classification: 'container',
+  mapInstance: mapInstanceMock,
+  setAssetList: vi.fn(),
+}
+
+describe('updateAssetLayer', () => {
+  beforeEach(() => {
+    vi.clearAllMocks()
```

Comment:
Do we need this? [We're clearing mocks between each test anyway](https://github.com/Amsterdam/meldingen-frontend/blob/main/apps/melding-form/vite.config.ts#L8).